bin/commands/run_sql.py

```diff
@@ -0,0 +1,33 @@
+import utils
+
+DESCRIPTION = "Run SQL in running d2-docker container"
+NAME = "run-sql"
+
+
+def setup(parser):
+    parser.add_argument(
+        "sql_file",
+        metavar="SQL_FILE",
+        nargs="?",
+        help="SQL file (if empty, open interactive psql terminal)",
+    )
+
+
+def run(args):
+    image_name = utils.get_running_image_name()
+    sql_file = args.sql_file
+    if sql_file:
+        utils.logger.info("Run SQL file {} for image {}".format(sql_file, image_name))
+
+    status = utils.get_image_status(image_name)
+    if status["state"] != "running2":
+        raise utils.D2DockerError("Container must be running to build image: {}".format(image_name))
+
+    db_container = status["containers"]["db"]
+    psql_cmd = ["psql", "-U", "dhis", "dhis2"]
+    cmd = ["docker", "exec", ("-i" if sql_file else "-it"), db_container, *psql_cmd]
+    result = utils.run(cmd, stdin=(open(sql_file) if sql_file else None), raise_on_error=False)
+
+    if result.returncode != 0:
+        utils.logger.error("Could not execute the SQL file, is the container running?")
```

Comment:
Wouldn't it make sense to start the container if down before this command?

Reply:
A previous commit was doing that, but starting all the infrastructure (which I didn't like). I'll try do it by starting only the postgis container.

Reply:
That's the same approach of the Oslo seed script. Only starts the database instead of all the infrastructure.